src/devtools/client/debugger/src/components/TestInfo/TestInfo.module.css

```diff
@@ -35,3 +35,16 @@
 .TestsuitesErrorColor {
   color: var(--testsuites-error-color);
 }
+
+#TestSteps {
```

Comment:
This doesn't seem to be applying to anything but let's use a class name here anyway